PWGCF/FemtoDream/femtoDreamPairTaskTrackV0.cxx

```diff
@@ -29,76 +30,89 @@
 #include "FemtoUtils.h"
 
 using namespace o2;
-using namespace o2::analysis::femtoDream;
+using namespace o2::soa;
 using namespace o2::framework;
 using namespace o2::framework::expressions;
-using namespace o2::soa;
+using namespace o2::analysis::femtoDream;
 
 namespace
 {
-static constexpr int nPart = 2;
+static constexpr int nTrack = 1;
+static constexpr int nV0Children = 2;
 static constexpr int nCuts = 5;
-static const std::vector<std::string> partNames{"PartOne", "PartTwo"};
+static const std::vector<std::string> TrackName{"Track"};
+static const std::vector<std::string> V0ChildrenName{"PosChild", "NegChild"};
 static const std::vector<std::string> cutNames{"MaxPt", "PIDthr", "nSigmaTPC", "nSigmaTPCTOF", "MaxP"};
-static const float cutsTable[nPart][nCuts]{
-  {4.05f, 1.f, 3.f, 3.f, 100.f},
-  {4.05f, 1.f, 5.f, 5.f, 100.f}};
-
-static const std::vector<float> kNsigma = {3.5f, 3.f, 2.5f};
-
+static const float cutsTableTrack[nTrack][nCuts]{
+  {4.05f, 0.75f, 3.f, 3.f, 100.f}};
```

Comment:
Check that you are getting the initialization that you wanted

Reply:
The initalization should be fine. The kNsigma vector has been replaced by a configurable.